python/ql/src/Security/CWE-377/InsecureTemporaryFile.ql

```diff
@@ -1,6 +1,6 @@
 /**
  * @name Insecure temporary file
- * @description Creating a temporary file using mktemp may be insecure.
+ * @description Creating a temporary file name may be insecure.
```

Comment:
Presumably there are some ways to create a temporary file securely, so perhaps: "Creating a temporary file using this method may be insecure."?

Reply:
Done.